src/Connection.cpp

```diff
@@ -6,6 +6,9 @@
 #include "murmur_pch.h"
 
 #ifdef Q_OS_WIN
+#	ifdef _INC_QOS2
```

Comment:
You should probably define _WIN32_WINNT before including windows.h or other windows headers then. In mumble_pch.hpp.